docs/user/tutorial/quickstart/java11.rst

```diff
@@ -0,0 +1,143 @@
+:Author: Torben Barsballe
+:Thanks: geotools-user list
+:Version: |release|
+:License: Creative Commons with attribution
+
+**********************
+ Java 11 Quickstart 
+**********************
+
+.. sectionauthor:: Torben Barsballe <tbarsballe@boundlessgeo.com>
+
+Welcome 
+=======
+
+GeoTools 21.0 and newer can be used in a Java 11 modular application. This requires a few additions on top of the regular tutorial application.
```

Comment:
should we say "jigsaw" modular application, since modules were introduced prior to Java 11 (and will exist for longer than Java 11).

Reply:
My only concern is people tend to have no idea what you're talking about when you say "Jigsaw" Maybe this will change as people get more familiar with it, but for now I prefer it as written since most people will just care about what they need to do to get it running on Java 11.